src/skimage/filters/edges.py

```diff
@@ -710,7 +714,9 @@ def laplace(image, ksize=3, mask=None):
         image = img_as_float(image)
     # Create the discrete Laplacian operator - We keep only the real part of
     # the filter
-    _, laplace_op = laplacian(image.ndim, (ksize,) * image.ndim)
+    _, laplace_op = laplacian(
+        image.ndim, shape=(ksize,) * image.ndim, connectivity=connectivity, sign=sign
+    )
     result = convolve(image, laplace_op)
```

Comment:
I just noticed that the `shape=...` argument seems to have no effect on the current implementation on `main`. 

https://github.com/scikit-image/scikit-image/blob/bd6cfaa4672e0291cb4e08bb1b2071705acad89e/src/skimage/restoration/uft.py#L442-L451

only uses `shape` to compute the transfer function, which isn't used here... So the function making that available is confusing?

Reply:
@glemaitre, do you remember more about this from 41875cf59ab2c638ab73a093aa269711b1bf165d? I currently think this was probably a misunderstanding.

So I think we can safely deprecate this argument?